components/post_view/message_attachments/message_attachment/message_attachment.jsx

```diff
@@ -198,7 +198,7 @@ export default class MessageAttachment extends React.PureComponent {
                     key={'attachment__field-caption-' + i + '__' + nrTables}
                     width='50%'
                 >
-                    {field.title}
+                    <Markdown message={field.title}/>
```

Comment:
this will render more than emojis, meaning the title of the message attachment include links, tables, etc.. they will also render, I think we should make it so only emojis are rendered.

Reply:
We ran into something similar with https://github.com/mattermost/mattermost-webapp/pull/2476 earlier, so you can take a look at that to see how they fixed this problem.